mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule BambooConfigAdapter.MixProject do
 
   defp deps do
     [
-      {:bamboo, "~> 0.8"},
+      {:bamboo, ">= 0.8.0 or < 2.0.0"},
```

Comment:
I think that we should use `and` since this would still match on version 2.0.0, and version 0.1.0. I can pull it in right after that, and I will cut a new release.